<?xml version="1.0" encoding="UTF-8"?>
<class xmlns="http://xml.phpdox.net/src" full="Symfony\Component\VarDumper\Exception\ThrowingCasterException" namespace="Symfony\Component\VarDumper\Exception" name="ThrowingCasterException" abstract="false" final="false" start="17" end="27">
  <file path="/home/web/example/src/freegame_developer/vendor/symfony/var-dumper/Exception" file="ThrowingCasterException.php" realpath="/home/web/example/src/freegame_developer/vendor/symfony/var-dumper/Exception/ThrowingCasterException.php" size="717" time="2016-10-06T08:40:01+00:00" unixtime="1475743201" sha1="6a1a6ee604040fcc9067e930a417e47d93cbf005" relative="freegame_developer/vendor/symfony/var-dumper/Exception/ThrowingCasterException.php"/>
  <docblock>
    <description/>
    <author value="Nicolas Grekas &lt;p@tchwork.com&gt;"/>
  </docblock>
  <extends full="Exception" namespace="" name="Exception"/>
  <constructor name="__construct" start="23" end="26" abstract="false" final="false" static="false" visibility="public">
    <docblock>
      <description/>
      <param description="The failing caster" type="object" variable="$caster">
        <type full="Symfony\Component\VarDumper\Exception\callable" namespace="Symfony\Component\VarDumper\Exception" name="callable"/>
      </param>
      <param description="The exception thrown from the caster" type="object" variable="$prev">
        <type full="\Exception" namespace="" name="Exception"/>
      </param>
    </docblock>
    <parameter name="caster" byreference="false" type="{unknown}"/>
    <parameter name="prev" byreference="false" type="object">
      <type full="Exception" namespace="" name="Exception"/>
    </parameter>
  </constructor>
</class>
